start-site/src/main/java/io/spring/start/site/extension/dependency/testcontainers/TestcontainersProjectGenerationConfiguration.java

```diff
@@ -80,20 +95,16 @@ BuildCustomizer<Build> springBootTestcontainersBuildCustomizer() {
 		}
 
 		@Bean
-		HelpDocumentCustomizer springBootTestcontainersHelpDocumentCustomizer(ProjectDescription description) {
-			return (helpDocument) -> {
-				String referenceDocUrl = String.format(
-						"https://docs.spring.io/spring-boot/docs/%s/reference/html/features.html#features.testing.testcontainers",
-						description.getPlatformVersion());
-				helpDocument.gettingStarted()
-					.addReferenceDocLink(referenceDocUrl, "Spring Boot Testcontainers support");
-			};
+		TestContainersHelpDocumentCustomizer springBootTestcontainersHelpDocumentCustomizer(
+				ProjectDescription description, ServiceConnections serviceConnections) {
+			return new TestContainersHelpDocumentCustomizer(description, serviceConnections);
 		}
 
 	}
 
 	@Configuration(proxyBeanMethods = false)
 	@ConditionalOnPlatformVersion("3.1.0-RC1")
+	@Import(ServiceConnectionsConfiguration.class)
```

Comment:
This second import seems to be redundant as the import on `SpringBootSupportConfiguration` will apply whenever this one would.